doc/modules/ensemble.rst

```diff
@@ -838,7 +817,163 @@ accessed via the ``feature_importances_`` property::
 
  * :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_regression.py`
 
- .. _voting_classifier:
+.. _histogram_based_gradient_boosting:
+
+Histogram-Based Gradient Boosting
+=================================
+
+Scikit-learn 0.21 introduces two new experimental implementations of
+gradient boosting trees, namely :class:`HistGradientBoostingClassifier`
+and :class:`HistGradientBoostingRegressor`, inspired by
+`LightGBM <https://github.com/Microsoft/LightGBM>`_.
+
+These histogram-based estimators can be **orders of magnitude faster**
+than :class:`GradientBoostingClassifier` and
+:class:`GradientBoostingRegressor` when the number of samples is larger
+than tens of thousands of samples.
+
+They also have built-in support for missing values, which avoids the need
+for an imputer.
+
+These fast estimators first bin the input samples ``X`` into
+integer-valued bins (typically 256 bins) which tremendously reduces the
```

Comment:
Will a user get confused here with `max_bins`?

Reply:
It's is detailed that one bin is reserved for missing values in the docstring.

Reply:
Later in this user guide it says:

> It is generally recommended to use as many bins as possible, which is the default: 255 bins.

which can be confusing.

Even over here we are binning `X` with 255 bins, since we are always saving one bind for missing.

Reply:
I removed the 255